dragn/dice/__init__.py

```diff
@@ -1,6 +1,22 @@
+import re
+
 from dragn.dice.die import DieBuilder
 
 
+# Module level __getattr__ to automatically create dice when
```

Comment:
This comment is great because this is not an obvious function, could you please move it into a docstring inside the function? It's a convention in Python and makes it easier for people to read it.

```python
def __getattr__(name: str):
    """
    Module level __getattr__ to automatically create dice when
    one a die of non-standard sizes.
    """
```